x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -0,0 +1,188 @@
+          }
+
+          // if searchbar is empty, filter to only applications and sort alphabetically
+          results = results.filter(({ type }: GlobalSearchResult) => type === 'application');
+
+          arr = [...results, ...arr].sort((a, b) => {
+            const titleA = a.title.toUpperCase(); // ignore upper and lowercase
+            const titleB = b.title.toUpperCase(); // ignore upper and lowercase
+            if (titleA < titleB) return -1;
+            if (titleA > titleB) return 1;
+            return 0;
+          });
+
+          setOptions(arr);
+        },
+        error: () => {
+          // TODO #74430 - add telemetry to see if errors are happening
+          // Not doing anything on error right now because it'll either just show the previous
+          // results or empty results which is basically what we want anyways
+        },
+        complete: () => {
+          ifMounted(() => setSearchValue(currentValue));
+        },
+      });
+    },
+    [globalSearch, searchValue, ifMounted]
+  );
+
+  const onWindowKeyDown = (event: KeyboardEvent) => {
+    if (event.key === '/' && (isWindows ? event.ctrlKey : event.metaKey)) {
+      if (searchRef) {
+        event.preventDefault();
+        searchRef.focus();
+      }
+    }
+  };
+
+  const onChange = (selected: EuiSelectableTemplateSitewideOption[]) => {
+    // @ts-ignore - ts error is "union type is too complex to express"
+    const { url } = selected.find(({ checked }) => checked === 'on');
+
+    if (/^https?:\/\//.test(url)) window.location.assign(url);
+    else {
+      navigateToUrl(url);
+      (document.activeElement as HTMLElement).blur();
+      onSearch('');
+      if (searchRef) {
+        searchRef.value = '';
+        searchRef.dispatchEvent(blurEvent);
+      }
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('keydown', onWindowKeyDown);
+
+    return () => {
+      window.removeEventListener('keydown', onWindowKeyDown);
+    };
+  });
+
+  useEffect(() => {
+    onSearch('');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <EuiSelectableTemplateSitewide
+      onChange={onChange}
+      options={options}
+      searchProps={{
+        onSearch,
+        'data-test-subj': 'header-search',
+        inputRef: setSearchRef,
+        compressed: true,
+        placeholder: i18n.translate('xpack.globalSearchBar.searchBar.placeholder', {
+          defaultMessage: 'Search Elastic',
+        }),
+      }}
+      popoverFooter={
+        <EuiText color="subdued" size="xs">
+          <EuiFlexGroup
+            alignItems="center"
+            justifyContent="flexEnd"
+            gutterSize="s"
+            responsive={false}
+            wrap
+          >
+            <FormattedMessage
+              id="xpack.globalSearchBar.searchBar.shortcut"
+              defaultMessage="{what}{how}"
+              values={{
+                what: <EuiFlexItem grow={false}>Quickly search using:</EuiFlexItem>,
```

Comment:
I'm double checking on this but I see other people doing this so I really hope this is ok...

Reply:
😬

Reply:
I think this is OK, but the "Quickly search using:" text won't be translated. To solve this I think you can just wrap it in `i18n.translate`. Also I noticed that the `defaultMessage` on line 173 doesn't contain any strings, so you don't need to use `FormattedMessage` as a parent component. I think you can do this instead:

```tsx
<EuiFlexItem grow={false}>
  {i18n.translate('xpack.globalSearchBar.searchBar.shortcutDescription', {
    defaultMessage: 'Quickly search using:',
  })}
</EuiFlexItem>

<EuiFlexItem grow={false}>
  <EuiBadge>{isWindows ? 'Ctrl + S' : 'Command + S'}</EuiBadge>
</EuiFlexItem>
```

Also I changed `xpack.globalSearchBar.searchBar.shortcut` to `xpack.globalSearchBar.searchBar.shortcutDescription`, to follow the i18n id-naming guidelines: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

Reply:
I wanted to keep these as one string for languages that may not have the same order as English allowing the translators to mix and match order or put in break/stop words appropriately into the whole phrase instead of only seeing each half.

Reply:
👍  Sounds good, but are you sure the "Shortcut", "Command + /", and "Control + /" strings are being internationalized? I believe the translators will only see the "{what}{how}" string, so they can change order and insert new words, but they can't localize the strings you're using. I'm pretty sure you need to decorate them with additional i18n.

Reply:
Yeah, you were right!

After a consult with @Bamieh, I fixed the translation stuff by nesting another `FormattedMessage` inside the `values`. 

Similar example findable in the [i18n docs](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types) by searching "For complex messages"